app/controllers/projects_controller.rb

```diff
@@ -62,6 +63,45 @@ def destroy
     }
   end
 
+  private def set_location_params
+    return unless RailsDrivers.loaded.include?(:client_location_history)
+
+    @locations = @project.enrollment_location_histories.where(located_on: location_filter.range)
+    @markers = @locations.map { |l| l.as_marker_with_name(current_user) }
+    @bounds = ClientLocationHistory::Location.bounds(@locations)
+    @options = {
+      bounds: @bounds,
+      cluster: true,
+      marker_color: '#72A0C1',
```

Comment:
Should we keep this hardcoded hex value for consistency? Should we put it in some common location? Matches with:
- `drivers/client_location_history/app/controllers/client_location_history/clients_controller.rb`
- `drivers/client_location_history/app/controllers/client_location_history/warehouse_reports/client_location_history_controller.rb`

Reply:
Probably worth moving it to a constant or class method on `ClientLocationHistory::Location` for now.  At least that will centralize it, and we could potentially move it to the DB if we decide we need to.  Maybe?
```
marker_color: ClientLocationHistory::Location::MARKER_COLOR
```